app/views/exchange_rates/_document_detail.html.erb

```diff
@@ -3,14 +3,18 @@
   <section class="attachment exchange-rates-attachment">
     <div class="attachment-thumb">
         <%= link_to(image_pack_tag("spreadsheet.png", alt: ""), asset_path('todo')) %>
-    
     </div>
     <div class="attachment-details">
         <h3 class="govuk-heading-s govuk-!-font-size-27 govuk-!-font-weight-regular exchange-rates-period-heading">
           <%= Date::MONTHNAMES[period.month] %> <%= period.year %> monthly exchange rates
         </h3>
 
-        <p><strong><a class="govuk-link" href="/todo">View online</a></strong></p>
+        <p><strong><a class="govuk-link" href="/todo">View online</a></strong> | 
+        <% period.files.each_with_index do |file, index| %>
+          <strong><a href="<%= file.file_path %>"><%= file.format.upcase %></strong> <%= file.file_size_in_kb %> KB</a>
+          <% unless index == period.files.size - 1 %>|<% end %>
```

Comment:
Wondering if we could put the pipe before the unless?

```suggestion
          <%= "|" unless index == period.files.size - 1 %>
```

Reply:
done